rustls/src/lib.rs

```diff
@@ -271,6 +271,19 @@
 //!
 //!   Note that aws-lc-rs has additional build-time dependencies like cmake.
 //!   See [the documentation](https://aws.github.io/aws-lc-rs/requirements/index.html) for details.
+//!
+//!
+//! # Unexpected EOF
+//!
+//! TLS has a [`close_notify` mechanism to prevent truncation attacks](https://datatracker.ietf.org/doc/html/rfc8446#section-6.1).
+//! According to the TLS RFCs, each party is required to send it before closing the write side of the connection.
+//! However, some implementations don't send it. So long as the application layer protocol (for instance HTTP/2)
+//! has message length framing and can reject truncated messages, this is not a security problem.
+//!
+//! Because rustls doesn't know the framing of the application layer above it, it treats an EOF without close_notify as an error,
```

Comment:
nit: consistent backticks for `close_notify`.
```suggestion
//! Because rustls doesn't know the framing of the application layer above it, it treats an EOF without `close_notify` as an error,
```